.stylelintrc.js

```diff
@@ -9,11 +10,24 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['polaris-react/**/*.scss'],
+      files: ['polaris-react/**/*.{css,scss}'],
+      ignoreFiles: ['polaris-react/build/**/*.{css,scss}'],
       extends: [
         '@shopify/stylelint-plugin/prettier',
         './stylelint-polaris/configs/internal',
       ],
     },
   ],
+  ignoreFiles: [
+    '.next/**/*.{css,scss}',
+    'node_modules/**/*.{css,scss}',
+    'dist/**/*.{css,scss}',
+    'documentation/guides/legacy-polaris-v8-public-api.scss',
+    'polaris-react/build/**/*.{css,scss}',
+    'polaris-react/build-internal/**/*.{css,scss}',
+    'polaris-tokens/dist/**/*.{css,scss}',
```

Comment:
Is this already covered by `'dist/**/*.{css,scss}'` above?

Reply:
It works with a [slight tweak](https://github.com/Shopify/polaris/pull/6445/commits/d610383abb0d07093df37fa122a8724a6bdf392e), thanks for the idea!